lib/core/list-different.ts

```diff
@@ -48,7 +48,7 @@ export const checkFile = (
 
       const path = getTypeDefinitionPath(file);
 
-      const content = fs.readFileSync(path, { encoding: "UTF8" });
+      const content = fs.readFileSync(path, { encoding: "utf8" });
```

Comment:
I can revert this if I need to. TypeScript wouldn't let me build the project locally without this change.

Reply:
Were you getting a type error, or run time error? The types that I have (at least locally) define this as a `string` so any string should be valid?